paper/paper.md

```diff
@@ -5,44 +5,52 @@ tags:
 - molecular-dynamics-simulation
 - order-parameters
 - lipids
-- united-atom 
+- united-atom
 authors:
 - name: Hubert Santuz
-  orcid: 
-  affiliation: 1
+  orcid: 0000-0001-6149-9480
+  affiliation: "1, 2"
 - name: Amélie Bacle
-  orcid: 
-  affiliation: 2
+  orcid: 0000-0002-3317-9110
+  affiliation: 3
 - name: Pierre Poulain
   orcid: 0000-0003-4177-3619
-  affiliation: 3
+  affiliation: 4
 - name: Patrick F.J. Fuchs^[corresponding author]
   orcid: 0000-0001-7117-994X
-  affiliation: "4, 5"
+  affiliation: "5, 6"
 affiliations:
-- Laboratoire de Biochimie Théorique (LBT), CNRS, F-75005 Paris, France
+- name: CNRS, Université de Paris, UPR 9080, Laboratoire de Biochimie Théorique, 13 Rue Pierre et Marie Curie, F-75005 Paris, France
   index: 1
-- Laboratoire Coopératif "Lipotoxicity and Channelopathies - ConicMeds", Université de Poitiers, F-86000 Poitiers, France
+- name: Institut de Biologie Physico-Chimique–Fondation Edmond de Rothschild, PSL Research University, Paris, France
   index: 2
-- name: Mitochondria, Metals and Oxidative Stress group, Institut Jacques Monod, UMR 7592, Université de Paris, CNRS, F-75013 Paris, France.
+- name: Laboratoire Coopératif "Lipotoxicity and Channelopathies - ConicMeds", Université de Poitiers, F-86000 Poitiers, France
   index: 3
-- Sorbonne Université, Ecole Normale Supérieure, PSL Research University, CNRS, Laboratoire des Biomolécules (LBM), F-75005 Paris, France
+- name: Mitochondria, Metals and Oxidative Stress group, Institut Jacques Monod, UMR 7592, Université de Paris, CNRS, F-75013 Paris, France.
   index: 4
-- Université de Paris, UFR Sciences du Vivant, F-75013 Paris, France
+- name: Sorbonne Université, Ecole Normale Supérieure, PSL Research University, CNRS, Laboratoire des Biomolécules (LBM), F-75005 Paris, France
   index: 5
+- name: Université de Paris, UFR Sciences du Vivant, F-75013 Paris, France
+  index: 6
 date: 27 May 2021
 bibliography: paper.bib
 ---
 
-# Statement of need
+# Background
+
+Molecular dynamics (MD) simulations of lipids are widely used to understand the complex structure and dynamics of biological or model membranes [@Tieleman1997; @Feller2000; @Lyubartsev2011]. They are very complementary to biophysical experiments and have thus become important to get insights at the microscopic scale. Many of them are performed using all-atom (AA) or united-atom (UA) representations. In AA force fields (such as CHARMM36 [@Klauda2010]), all the atoms are considered whereas in UA force fields (such as Berger [@Berger1997]) the aliphatic hydrogen atoms (Hs) are merged to their parent carbon into a larger particule representing a CH, CH2 or CH3 (e.g. a methyl group is represented by a single CH3 particle). In simulations of phospholipids, the use of UA representations allows to divide by almost 3 the number of atoms to simulate due to aliphatic chains of the lipids and reduce the computation time.
```

Comment:
On the last sentence, I prefer the phrasing before, I find `due to aliphatic chains of the lipids` unclear. My suggestion for this sentence: `In simulations of phospholipids, the use of UA representations allows to divide by almost 3 the number of atoms to simulate because these molecules contain many aliphatic Hs. It thus reduces the computational cost without loosing important chemical details.`

Reply:
okay, the suggestion is good!